app/windows/sign.js

```diff
@@ -0,0 +1,23 @@
+const { sign } = require('./codesign');
+
+exports.default = async function (configuration) {
+  const esrpTool = process.env.ESRP_PATH;
+  if (!esrpTool) {
+    // Ignore if the signing tool is not available.
```

Comment:
nit: Probably a warning here would be good.

``console.log(`Warning: exiting because esrpTool doesn't exist at: ${esrpTool}`)``

Reply:
Thing is that this is called per file, so we'd see dozens of such warnings. This is purposedly silent as users may not have the tool around.

Reply:
Ok, understood. I guess the action will fail if the tool isn't downloaded/there, so we'd be aware of that.

Reply:
The action has an option for using signing or not. But if it's required to use and there's no tool, it shall fail.